tests/installation/bootloader_svirt.pm

```diff
@@ -330,7 +330,7 @@ sub run {
 
     # connects to a guest VNC session
     select_console('sut', await_console => 0);
-    vmware_set_permanent_boot_device('cdrom');
+    vmware_set_permanent_boot_device($boot_device);
```

Comment:
Seems my yesterday's review comment is not posted successfully. This change LGTM, but I have concerns on impact. Other teams also have tests on vmware, with such default boot device change, will it influence their test? Please evaluate it. Better also involve them to review this PR, maybe they need extra validation from your side.

Reply:
Hello, could you please run VR with https://openqa.nue.suse.com/tests/5991037

Reply:
Hi, I kicked off a VR and got passed - https://openqa.nue.suse.com/tests/6081225